src/makeCalculator.js

```diff
@@ -4,7 +4,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+
+      return this;
+    },
+    subtract(number) {
+      this.result -= number;
+
+      return this;
+    },
+    multiply(number) {
+      this.result *= number;
+
+      return this;
+    },
+    divide(number) {
+      this.result /= number;
+
+      return this;
+    },
+    operate(callback, number) {
```

Comment:
The 'operate' method correctly uses the provided callback and applies it to the 'result' property. It also allows for chaining by returning 'this'. Make sure to validate the input or handle potential errors when using callbacks.